src/utils/api.js

```diff
@@ -19,6 +19,10 @@ if (isProd) {
     if (!endpoint) {
         endpoint = `${url}/api`
     }
+} else {
+    // for dev. environments
```

Comment:
We do the same above: https://github.com/dhis2/ui/blob/a72a485ef98733bb72aeeb07bd53852926bcc052/src/utils/api.js#L13-L15

Maybe we should remove the L13-L15 block as `DHIS_CONFIG` only exists in Dev?

Reply:
re-written for better readability

```js
const IS_PROD = process.env.NODE_ENV === 'production'

let url
let endpoint

if (IS_PROD) {
    url = '..'
    endpoint = `${url}/api`
    if (typeof manifest !== 'undefined') {
        let version = manifest.dhis2.apiVersion
        url = manifest.activities.dhis.href
        endpoint = `${url}/api/${version}`
    }
} else {
    // for dev. environments
    url = DHIS_CONFIG.baseUrl
    endpoint = `${url}/api`
}
```